gatsby-site/src/components/SunriseFooter.js

```diff
@@ -0,0 +1,120 @@
+    </defs>
+  </svg>
+);
+
+const emailIcon = (
+  <svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path fillRule="evenodd" clipRule="evenodd" d="M18 36C27.9411 36 36 27.9411 36 18C36 8.05887 27.9411 0 18 0C8.05887 0 0 8.05887 0 18C0 27.9411 8.05887 36 18 36ZM8.5 11.6371H7.60484V12.5322V24.4677V25.3629H8.5H27H27.8952V24.4677V12.5322V11.6371H27H8.5ZM9.39516 23.5725V14.3947L16.8924 20.3925L17.4339 20.8257L17.9887 20.4096L26.1048 14.3225V23.5725H9.39516ZM11.0519 13.4274H24.3145L17.4693 18.5613L11.0519 13.4274Z" fill="white"/>
+  </svg>
+);
+
+const iconForSocialMediaType = (socialMediaType) => {
+  switch (socialMediaType) {
+    case 'facebook':
+      return fbIcon;
+    case 'instagram':
+      return igIcon;
+    case 'email':
+      return emailIcon;
+    default:
+  }
+  return null;
+}
+
+const SunriseFooter = ({ hubName, socialMediaMap }) => {
+  if (hubName == null || socialMediaMap == null) {
+    return null;
+  }
+  
+  const socialMediaComponents = [];
+
+  for (const [socialMediaType, link] of socialMediaMap.entries()) {
+    if (link == null) {
+      continue;
+    }
+    socialMediaComponents.push(
+      <div key={socialMediaType}>
+          <ImageCircle href={link} key={socialMediaType}>
+            <ImageStyle>
+              {iconForSocialMediaType(socialMediaType)}
+            </ImageStyle>
+          </ImageCircle>
+      </div>
+    );
+  }
+
+  return <Container>
+    <Button label={`Donate to Sunrise ${hubName}`} href="https://secure.actblue.com/donate/sunrisemvmt" />
```

Comment:
Ah I think I have to add padding under the button according to Figma...can do that tomorrow

Reply:
Everything else looks good so if you want to merge this without the padding we can come back to it in another PR